src/app/containers/BulletedListItem/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import Blocks from '../Blocks';
+import paragraph from '../Paragraph';
+import { listItemPropTypes } from '#models/propTypes/list';
+
+const componentsToRender = { paragraph };
+
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+
+const BulletedListItemContainer = ({ blocks }) => {
+  return (
+    <li>
+      <Wrapper>
+        <Blocks blocks={blocks} componentsToRender={componentsToRender} />
+      </Wrapper>
+    </li>
+  );
+};
+
+BulletedListItemContainer.propTypes = { ...listItemPropTypes };
```

Comment:
Is it possible to just put the prop types here?  I get this is an existing pattern for the article blocks though so happy if not - would be nice to simplify if possible though

Reply:
Done